### PR TITLE
Deployment server fix

### PR DESCRIPTION
**Pull request by bretm9**
### Participating Group Members:

- <name>
- <name>

### Code Highlights:

- Adds static.json
- Updates scripts in package.json to deploy on Heroku

### Have Tests Been Added?

- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?
This branch needs to be merged with main before we can see it up and running in Heroku, therefore we can't verify that this branch will deploy properly before merge.

### Issues:

- Addresses #48 
- Closes #

### Screenshots (if appropriate):
N/A

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: node src/server.js
```
Comment (nicolegooden) [inline Procfile:1]: Should we remove this Procfile?

**Review by nicolegooden — APPROVED** [review]
Thanks, Bret!  Hoping we are one step closer to deployment.

**Comment by npdarrington** [comment]
Thank you Bret! Hopefully this will work!

**Review by npdarrington — APPROVED** [review]

**Review by stacyp2006 — APPROVED** [review]
Hoping this works!

**Comment by bretm9** [comment]
I had a left over Procfile from when I was attempting to build with an express server. This has now been removed.

**Pull request merged**